### ISSUE TITLE
Readme triggers firefox bug

### ISSUE THREAD

**Issue by Chronial** [wallix/awless#170]
Open the readme in the current version of firefox (57.0.1) causes a massive expansion in firefox memory usage. Screenshot by Hacker News user satysin:

![screenshot](https://i.imgur.com/KJaWy2G.png)

This has been reported on linux and windows: https://news.ycombinator.com/item?id=15845396

While this is probably not your fault, but a bug in firefox, you might still want to do something about. At the moment visiting your github page can cause system crashes ^^.

**Comment by fxaguessy** [comment]
Thanks for the report.
We need to investigate why this happens and if it is a known bug in Firefox.

**Comment by fxaguessy** [comment]
I confirm that it is the same on macOS.

**Comment by hbi-wallix** [comment]
Could be APNG? But it shouldn't be of course, it is a great format highly efficient for screencasts.

**Comment by fxaguessy** [comment]
Yes, it is APNG (loading [https://raw.githubusercontent.com/wiki/wallix/awless/apng/awless-demo.png](https://raw.githubusercontent.com/wiki/wallix/awless/apng/awless-demo.png) causes the same problem). Maybe an implementation issue in Firefox.
On Chrome (v65), it requires 350Mo to display the APNG file.

**Comment by Chronial** [comment]
Maybe get grid of the APNG? I don't think a github readme should consume 350Mb of Memory :).
